permission_handler_apple/ios/Classes/strategies/LocationPermissionStrategy.m

```diff
@@ -108,7 +108,6 @@ - (void)locationManager:(CLLocationManager *)manager didChangeAuthorizationStatu
     }
     
     if (status == kCLAuthorizationStatusNotDetermined) {
-        _permissionStatusHandler(PermissionStatusDenied);
```

Comment:
Can't we make an additional PermissionStatus "NotDetermined"?

Reply:
The problem here is that iOS will emit `kCLAuthorizationStatusNotDetermined` when the permission dialog is opened. We do not want to call the callback at all until we capture the user's response.